### PR TITLE
Update Oregon.md

### LINKED ISSUE #1267: Incident in Portland, Oregon
## Location

Portland, Oregon

## Date

August 1st

## Description

Police charge, tackle, pepper spray and arrest numerous protestors

## Links

https://twitter.com/hungrybowtie/status/1289791877223350273
https://twitter.com/hungrybowtie/status/1289792999291985922
https://twitter.com/hungrybowtie/status/1289794603244072961
https://twitter.com/PDocumentarians/status/1289792025580023808
https://twitter.com/45thabsurdist/status/1289792288533577729
https://twitter.com/IwriteOK/status/1289787851027038208
https://www.youtube.com/watch?v=nwOZZxmTKeo
https://www.reddit.com/r/PublicFreakout/comments/i2hb9o/portland_police_order_protestors_to_continue/

### LINKED ISSUE #1268: Incident in Portland, Oregon
## Location

Portland, Oregon

## Date

August 1st

## Description

Tires of car are slashed by police as they walk by. The woman driving the car is wearing a helmet and PRESS shirt

## Links

https://twitter.com/hungrybowtie/status/1289793915332079617
[2:00](https://www.pscp.tv/w/1MnGndnYllExO)

### LINKED ISSUE #1269: Incident in Portland, Oregon
## Location

Portland, Oregon

## Date

August 1st

## Description

Police shove and pepper spray protestors obeying LRAD directions

## Links

https://twitter.com/TheRealCoryElia/status/1289789570947543040

### LINKED ISSUE #1270: Incident in Portland, Oregon
## Location

Portland, Oregon

## Date

August 1st

## Description

Police charge at and shove protestors.

## Links

https://twitter.com/hungrybowtie/status/1289801996799578114

### LINKED ISSUE #1271: Incident in Portland, Oregon
## Location

Portland, Oregon

## Date

August 1st

## Description

Journalist shoved into car and pepper sprayed.

## Links

https://twitter.com/FancyJenkins/status/1289808596243374082
https://twitter.com/hungrybowtie/status/1289807143961427970

### PR DESCRIPTION
**Pull request by emarcey**
Closes #1267 
Closes #1268 
Closes #1269
Closes #1270 
Closes #1271

### CODE REVIEW

File: reports/Oregon.md [hunk 1]
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
...
+
+
+
+
+https://twitter.com/FancyJenkins/status/1289808596243374082
+https://twitter.com/hungrybowtie/status/1289807143961427970
+
+
+
+
```
Comment (aaaaaron-ship-it) [inline reports/Oregon.md:3540]: Needs tag, Links heading, and more descriptive title.

File: reports/Oregon.md [hunk 2]
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
 * https://twitter.com/hungrybowtie/status/1288734291334053888
 
 
+### Police charge, tackle, and pepper spray protesters | August 1st
+
+Police gave directions to protesters to walk in a certain direction. Protesters followed directions. Police charged at protestors, shoving many to the ground. One protester on a bicycle is knocked over. The protester lands on top of the bike and has trouble getting up. Police pepper spray protesters as well.
```
Comment (aaaaaron-ship-it) [inline reports/Oregon.md:3480]: Might be a good idea to make it explicit that there were several instances. Also I think some details could be added (see below)

File: reports/Oregon.md [hunk 3]
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
...
+
+tags: protester, tackle, shove, grab, strike, pepper-spray, spray
+
+id: or-portland-225
+
+**Links**
+
+* https://twitter.com/hungrybowtie/status/1289791877223350273
```
Comment (aaaaaron-ship-it) [inline reports/Oregon.md:3488]: It's really tough to see, but I think a cop maces a protester on the ground here. Might want to mention this if that's what's happening.

File: reports/Oregon.md [hunk 4]
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
...
+* https://twitter.com/hungrybowtie/status/1289792999291985922
+* https://twitter.com/hungrybowtie/status/1289794603244072961
+* https://twitter.com/PDocumentarians/status/1289792025580023808
```
Comment (aaaaaron-ship-it) [inline reports/Oregon.md:3491]: Should probably mention the woman who is shoved to the ground, forced to get up and then shoved repeatedly while she tells officers she can't see.

File: reports/Oregon.md [hunk 5]
```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
...
+* https://twitter.com/45thabsurdist/status/1289792288533577729
```
Comment (aaaaaron-ship-it) [inline reports/Oregon.md:3492]: Might want to mention the person shoved to the ground who can't get up.

File: reports/Oregon.md (hunk 1 above)
Comment (emarcey) [reply reports/Oregon.md:3540]: Yeah this just got goofed up by the formatter

File: reports/Oregon.md (hunk 5 above)
Comment (emarcey) [reply reports/Oregon.md:3504]: Yeah this is the "person who has trouble getting up" there are several angles of the same thing here

**Comment by emarcey** [comment]
@aaaaaron-ship-it addressed some of your comments. I don't think 225 is really "multiple incidents" tho. as far as I can tell it's just a bunch of angles of the same thing.

File: reports/Oregon.md (hunk 5 above)
Comment (aaaaaron-ship-it) [reply reports/Oregon.md:3508]: Ah I didn't see.

**Pull request merged**